src/main/java/ch/njol/skript/events/SimpleEvents.java

```diff
@@ -741,6 +742,14 @@ public class SimpleEvents {
 				)
 				.since("2.7");
 
+		Skript.registerEvent("Unknown Command", SimpleEvent.class, UnknownCommandEvent.class, "[player] (wrong|unknown) (cmd|command) [send|use]")
```

Comment:
Must check if `UnknownCommandEvent` class exists (paper only)

Reply:
but the event is bukkit not paper iirc

Reply:
it's paper only, if you search for it in [Spigot's javadocs](https://hub.spigotmc.org/javadocs/bukkit/) you won't find it

Here is the [paper javadocs](https://jd.papermc.io/paper/1.20/org/bukkit/event/command/UnknownCommandEvent.html) for it